src/routes/RootRouter/RootRouter.js

```diff
@@ -33,23 +33,7 @@ export default function RootRouter () {
 
   if (isAuthorized) {
     return (
-      <Route
-        path={[
-          // The routes matching here serves to provide `match.params` to `AuthLayoutRouter`.
-          // It likely better to move away from this, pushing the related routes and features
-          // into their own components below `AuthLayoutRouter` receiving the `match.params`
-          // also already happening from the routes found there. Search for the following
-          // comment to see the places where the `match.params` supplied here are currently
-          // replied-upon:
-          //
-          // "NOTE: Relies on incoming `match.params` (currently provided by `RootRouter`)"
-          //
-          '/:context(groups)/:groupSlug/:view(events|groups|map|members|projects|settings|stream|topics)?',
```

Comment:
nice, what did you do?

Reply:
Well basically just moved it back into `AuthLayoutRouter` to approx. what you were doing before, but with nailed it down more. There were already a couple other `matchPaths` going on there still which I consolidated all into one place and put all (one exception) the flags dependent on this preliminary route matching (`pathMatchParams`) right next to it in the component. The idea here is that as we move forward we should be reducing and eventually mostly eliminating this pre-match in deference to pushing things into their own child router components... Something like that:

https://github.com/Hylozoic/hylo-evo/blob/1987e1c15e5cbe6e5fc96f53e1916ae81c5fac7a/src/routes/AuthLayoutRouter/AuthLayoutRouter.js#L66-L77